# Desired action specification
# string specifies the type of action

#######################
# Currently implimented
#######################
# 'depth_change'
# 'heading_change'
# 'set_rcvel'
# 'arm'
# 'rc_off'
string actionID
# common specifications, may be ignored depending on message type
float64 target_depth
float64 target_heading
int64 x_rc_vel
int64 y_rc_vel
bool arm
---

# Sucesseful result
string actionID
float64 end_depth
float64 end_heading
bool arm

---

# feedback message
string actionID
float64 current_depth
float64 current_heading
